src/content/en/2021/media.md

```diff
@@ -1,20 +1,900 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Media
-description: TODO
+description: Media chapter of the 2021 Web Almanac covering how images and videos are currently being encoded, embedded, styled, and delivered on the web
 authors: [eeeps, dougsillars]
-reviewers: []
-analysts: [eeeps, dougsillars]
+reviewers: [Navaneeth-akam, tpiros, akshay-ranganath, addyosmani]
+analysts: [eeeps, dougsillars, akshay-ranganath]
```

Comment:
@eeeps are these all up to date and have all the reviewers reviewed the draft (either in Google Docs or here)?

Reply:
Yes, these are all up to date and all of the reviewers listed here contributed comments / helped shape the final doc.